RegenAllByteCodeNoBuild.cmd

```diff
@@ -0,0 +1,62 @@
+::-------------------------------------------------------------------------------------------------------
+:: Copyright (C) Microsoft. All rights reserved.
+:: Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+::-------------------------------------------------------------------------------------------------------
+
+:: WARNING: be careful when using this script as it assumes that
+:: you already have bytecode-format-compatible builds for all required flavors.
+:: This script helps speed things up when you are only making changes to
+:: e.g. Intl.js, without making any changes to bytecode format, since
+:: rebuilding every flavor when there are no relevant changes is a waste of time.
+:: Please ensure that you use buddy builds to validate the results.
+
+:: Regenerate all bytecode (without rebuilding each flavor of ch.exe)
+:: ch.exe is used to generate Intl bytecodes.
+:: ch.exe (NoJIT variety) is used to generate NoJIT Intl bytecodes.
+:: Each set of bytecode requires an x86_debug and x64_debug binary.
+::
+:: Thus we need to already have compatible builds of the following:
+:: [Core]   ch.exe        x64_debug
+:: [Core]   ch.exe        x86_debug
+:: [Core]   ch.exe        x64_debug (NoJIT)
+:: [Core]   ch.exe        x86_debug (NoJIT)
+
+@echo off
+
+::main
+  setlocal
+    set _reporoot=%~dp0
+    pushd %_reporoot%
+      call :RegenByteCodeNoBuild x64
+      call :RegenByteCodeNoBuild x64 nojit
+      call :RegenByteCodeNoBuild x86
+      call :RegenByteCodeNoBuild x86 nojit
+    popd
+  endlocal
+  goto :eof
+
+:RegenByteCodeNoBuild
+  setlocal
+    set _arch=%1
+    set _bits=
+    if [%_arch%] == [x64] (
+      set _bits=64b
+    ) else if [%_arch%] == [x86] (
+      set _bits=32b
+    )
+
+    set _nojit=
+    if [%2] == [nojit] (
+      set _nojit=.nojit
+    )
+
+    pushd lib\Runtime\Library\InJavascript
+      set _script=Intl.js
+      set _header=%_script%%_nojit%.bc.%_bits%.h
+      set _bin=%_reporoot%\Build\VcBuild%_nojit%\bin\%_arch%_debug\ch.exe
+
+      echo %_bin% -GenerateLibraryByteCodeHeader:%_header% -Intl %_script%

```

Comment:
Ah good point. I thought the GenByteCode.cmd script forced a build. Made the change.